axoproject/src/rust.rs

```diff
@@ -89,6 +87,23 @@ fn workspace_info(pkg_graph: &PackageGraph) -> Result<WorkspaceStructure> {
     let target_dir = workspace.target_directory().to_owned();
     let workspace_dir = workspace.root().to_owned();
 
+    let mut axoupdater_version = None;
```

Comment:
Ok so this is *probably* fine but strictly speaking it's not precise enough. In an ideal world we compute this value *per-package* (so construct a Map of Package -> Opt<Version>). That way if a package we don't care about "randomly" depends on axoupdater we don't freak out.